core/src/main/java/org/elasticsearch/search/suggest/SuggestionBuilder.java

```diff
@@ -212,6 +223,8 @@ protected void populateCommonFields(MapperService mapperService,
                 throw new IllegalArgumentException("Analyzer [" + luceneAnalyzer + "] doesn't exists");
```

Comment:
This should be `analyzer` not `luceneAnalyzer`? or we don't report the name of the analyzer that doesn't exist